django/contrib/sitemaps/__init__.py

```diff
@@ -47,7 +48,12 @@ def _get_sitemap_full_url(sitemap_url):
 
     Site = django_apps.get_model('sites.Site')
     current_site = Site.objects.get_current()
-    return 'http://%s%s' % (current_site.domain, sitemap_url)
+    if not site_domain:
+        site_domain = current_site.domain
```

Comment:
lines 49 and 50 should be under the `if` too, otherwise `sites` is still being used

Reply:
Yeah, good catch! Thank you :)